stratum/hal/lib/tdi/es2k/es2k_sde_meter.cc

```diff
@@ -98,7 +98,7 @@ ::util::Status Es2kSdeWrapper::WritePktModMeter(
   return ::util::OkStatus();
 }
 
-::util::Status GetMeterField(
+static ::util::Status GetMeterField(
```

Comment:
Does this need to be static or can this be added to the class?

Reply:
1. It's a standalone function. It doesn't reference any data members of the class.
3. It shouldn't be added to the class because it's an implementation detail. We don't want to expose its existence in the header file.
4. gcc issues a warning because it's a global function with no function prototype.
5. It's not used outside this file, so it should be static.
6. I've used the same pattern to address the problem in previous commits.